### PR TITLE
Change class loading and parsing warnings to debug messages.

### PR DESCRIPTION
**Pull request by jeannielynnmoulton**
Often, there are so many of these warnings that they lose all meaning as
warning, so they are now moved to debug.

- [x] Each commit message has a non-empty body, explaining why the change was made.
- [x] Methods or procedures I have added are documented, following the guidelines provided in CODING_STANDARD.md.
- [x] The feature or user visible behaviour I have added or modified has been documented in the User Guide in doc/cprover-manual/
- [x] Regression or unit tests are included, or existing tests cover the modified code (in this case I have detailed which ones those are in the commit message).
- [x] My commit message includes data points confirming performance improvements (if claimed).
- [x] My PR is restricted to a single feature or bugfix.
- [x] White-space or formatting changes outside the feature-related changed lines are in commits of their own.

### CODE REVIEW

File: jbmc/src/java_bytecode/java_bytecode_convert_class.cpp [hunk 1]
```diff
@@ -302,7 +302,7 @@ void java_bytecode_convert_classt::convert(
     }
     catch(const unsupported_java_class_signature_exceptiont &e)
     {
-      warning() << "Class: " << c.name
+      debug() << "Class: " << c.name
```
Comment (allredj) [inline jbmc/src/java_bytecode/java_bytecode_convert_class.cpp:305]: What's the verbosity level for debug?

**Review by allredj — APPROVED** [review]

**Review by majakusber — APPROVED** [review]

File: jbmc/src/java_bytecode/java_bytecode_convert_class.cpp (hunk 1 above)
Comment (jeannielynnmoulton) [reply jbmc/src/java_bytecode/java_bytecode_convert_class.cpp:305]: it's like this
1 - error
2 - warning
4 - result
6 - status
8 - stats
10 - debug

**Review by thk123 — APPROVED** [review]
I think this makes sense, I wonder if we should still produce one `warning()`:

```
Unsupported java signatures were encountered. Some classes may be wrongly treated as not generic
```

If one of more warning would have previously been emitted.

**Comment by majakusber** [comment]
# [Codecov](https://codecov.io/gh/diffblue/cbmc/pull/5224?src=pr&el=h1) Report
> Merging [#5224](https://codecov.io/gh/diffblue/cbmc/pull/5224?src=pr&el=desc) into [develop](https://codecov.io/gh/diffblue/cbmc/commit/be84cd1e7e54f0d68930a480815e4971fe4ab8cd&el=desc) will **not change** coverage by `%`.
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/diffblue/cbmc/pull/5224?src=pr&el=tree)

```diff
@@           Coverage Diff            @@
##           develop    #5224   +/-   ##
========================================
  Coverage    67.37%   67.37%           
========================================
  Files         1157     1157           
  Lines        95087    95087           
========================================
  Hits         64068    64068           
  Misses       31019    31019           
```

| Flag | Coverage Δ | |
|---|---|---|
| #cproversmt2 | `42.65% <ø> (ø)` | :arrow_up: |
| #regression | `63.89% <66.66%> (ø)` | :arrow_up: |
| #unit | `31.92% <88.88%> (ø)` | :arrow_up: |


------

[Continue to review full report at Codecov](https://codecov.io/gh/diffblue/cbmc/pull/5224?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/diffblue/cbmc/pull/5224?src=pr&el=footer). Last update [be84cd1...979fb9d](https://codecov.io/gh/diffblue/cbmc/pull/5224?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

**Pull request merged**